src/windows/winjector-helper-core.c

```diff
@@ -380,18 +382,27 @@ initialize_remote_worker_context (RemoteWorkerContext * rwc,
   /* } */
   gum_x86_writer_put_label (&cw, skip_unload_label);
 
+  /* set return 0 */
+  gum_x86_writer_put_xor_reg_reg(&cw, GUM_REG_EAX, GUM_REG_EAX);
+
+  /* if (call GetLastError) { */
+  gum_x86_writer_put_jcc_short_label(&cw, X86_INS_JE, restore_registers, GUM_NO_HINT);
+  gum_x86_writer_put_label(&cw, loadlibrary_failed_label);
+  gum_x86_writer_put_call_reg_offset_ptr_with_arguments (&cw, GUM_CALL_SYSAPI,
+      GUM_REG_XBX, G_STRUCT_OFFSET(RemoteWorkerContext, get_last_error_impl),
+      1,
```

Comment:
This should be 0 and the next two arguments omitted. (`GetLastError()` doesn't expect any arguments.)

Reply:
done.